src/coreclr/jit/codegenarm64.cpp

```diff
@@ -10104,6 +10107,232 @@ void CodeGen::genArm64EmitterUnitTests()
 
 #endif
 
+#ifdef ALL_ARM64_EMITTER_UNIT_TESTS_SVE
+    //
+    // R_R_R   SVE operations, one dest, two source
+    //
+
+    genDefineTempLabel(genCreateTempLabel());
+
+    theEmitter->emitIns_R_R_R(INS_sve_and, EA_SCALABLE, REG_V0, REG_P1, REG_V2,
+                              INS_OPTS_SCALABLE_B); // AND     <Zdn>.<T>, <Pg>/M, <Zdn>.<T>, <Zm>.<T>
+    theEmitter->emitIns_R_R_R(INS_sve_bic, EA_SCALABLE, REG_V3, REG_P4, REG_V5,
+                              INS_OPTS_SCALABLE_H); // BIC     <Zdn>.<T>, <Pg>/M, <Zdn>.<T>, <Zm>.<T>
+    theEmitter->emitIns_R_R_R(INS_sve_eor, EA_SCALABLE, REG_V14, REG_P5, REG_V16,
+                              INS_OPTS_SCALABLE_S); // EOR     <Zdn>.<T>, <Pg>/M, <Zdn>.<T>, <Zm>.<T>
+    theEmitter->emitIns_R_R_R(INS_sve_orr, EA_SCALABLE, REG_V29, REG_P7, REG_V31,
+                              INS_OPTS_SCALABLE_D); // ORR     <Zdn>.<T>, <Pg>/M, <Zdn>.<T>, <Zm>.<T>
+
+    theEmitter->emitIns_R_R_R(INS_sve_add, EA_SCALABLE, REG_V5, REG_P6, REG_V7,
+                              INS_OPTS_SCALABLE_B); // ADD     <Zdn>.<T>, <Pg>/M, <Zdn>.<T>, <Zm>.<T>
+    theEmitter->emitIns_R_R_R(INS_sve_sub, EA_SCALABLE, REG_V15, REG_P7, REG_V29,
+                              INS_OPTS_SCALABLE_H); // SUB     <Zdn>.<T>, <Pg>/M, <Zdn>.<T>, <Zm>.<T>
+    theEmitter->emitIns_R_R_R(INS_sve_subr, EA_SCALABLE, REG_V2, REG_P0, REG_V13,
+                              INS_OPTS_SCALABLE_S); // SUBR    <Zdn>.<T>, <Pg>/M, <Zdn>.<T>, <Zm>.<T>
+
+    theEmitter->emitIns_R_R_R(INS_sve_sdiv, EA_SCALABLE, REG_V3, REG_P2, REG_V9,
```

Comment:
The tool generated just 1 line per instruction, but we should add some more permutation for registers, `emitAttr` and `instOpts` to make sure `emitIns_*` methods works as expected.

Reply:
For each small group I've made sure to try to go across as many permutations as possible. (ie: `INS_sve_and, INS_sve_bic, INS_sve_eor, INS_sve_orr` are all `IF_SVE_AA_3A` and go through the same code so will have the same boundaries). The number of instructions here meant I felt there was decent coverage. I also made sure to hit value boundaries where possible (0, 31 etc).

The only instruction I felt I needed to add more permutations on were the `EA_4BYTE` versions of `clasta/clastb`.

What I found limiting is we can't test for error cases. Eg: I can't check that I can't generate a `B` version of `INS_sve_sdiv`.

Reply:
> Eg: I can't check that I can't generate a B version of INS_sve_sdiv.

Yes, I don't think we have code that verifies that something is not possible to get generated.